Template.yaml

```diff
@@ -0,0 +1,648 @@
+      TLS_DHE_DSS_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 
+      ,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256,
+      TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_DSS_WITH_AES_128_GCM_SHA256 
+      ,TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA,TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,SSL_RSA_WITH_3DES_EDE_CBC_SHA,
+      TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA,TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA,SSL_DHE_RSA_WITH_3DES_EDE_CBC_SHA,
+      SSL_DHE_DSS_WITH_3DES_EDE_CBC_SHA,TLS_EMPTY_RENEGOTIATION_INFO_SCSV"
+
+      sslVerifyClient="optional"
+
+
+      ["b7a.users"]
+
+      ["b7a.users.generalUser1"]
+
+      password="5BAA61E4C9B93F3F0682250B6CF8331B7EE68FD8"
+
+
+      [validationConfig]
+
+      enableRequestValidation = false
+
+      enableResponseValidation = false
+
+
+
+      [throttlingConfig]
+
+      enabledGlobalTMEventPublishing = false
+
+      jmsConnectioninitialContextFactory = "bmbInitialContextFactory"
+
+      jmsConnectionProviderUrl=
+      "amqp://admin:admin@carbon/carbon?brokerlist='tcp://localhost:5672'"
+
+      jmsConnectionUsername = ""
+
+      jmsConnectionPassword = ""
+
+      throttleEndpointUrl = "https://localhost:9443/endpoints"
+
+      throttleEndpointbase64Header = "admin:admin"
+
+
+      [tokenRevocationConfig]
+        [tokenRevocationConfig.realtime]
+          enableRealtimeMessageRetrieval = false
+          jmsConnectionTopic = "tokenRevocation"
+          jmsConnectioninitialContextFactory = "bmbInitialContextFactory"
+          jmsConnectionProviderUrl= "amqp://admin:admin@carbon/carbon?brokerlist='tcp://localhost:5672'"
+          jmsConnectionUsername = ""
+          jmsConnectionPassword = ""
+        [tokenRevocationConfig.persistent]
+          enablePersistentStorageRetrieval = false
+          useDefault = true
+          hostname = "https://127.0.0.1:2379/v2/keys/jti/"
+          username = "root"
+          password = "root"
+
+      [httpClients]
+        verifyHostname=true
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: microgateway-logging
+  data:
+    init.sh: >-
+      #!/bin/bash
+
+      /home/ballerina/wso2/runtime/bin/ballerina run --config /home/ballerina/conf/micro-gw.conf ${PROJECT_NAME}.balx
+
+#################################################################
+# Uncomment this section if you use logstash cluster
+#################################################################
+# - apiVersion: v1
+#  kind: Secret
+#  metadata:
+#    name: logstash-elasticsearch-creds
+#  type: Opaque
+#  data:
+#    password: ${{LOGSTASH_PASSWORD}}
+#    username: ${{LOSTASH_USERNAME}}
+
+# - apiVersion: v1
+#   kind: ConfigMap
+#   metadata:
+#     name: logstash-conf
+#   data:
+#     logstash.conf: |
+#       # Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's remove this license header from this since it's should only be defined once for the file.